app/components/authenticate/authenticate.component.ts

```diff
@@ -41,52 +37,4 @@ export class AuthenticateComponent implements OnInit {
   authenticateGithub(): void {
     authenticateUser(switchToAddRepositoryPanel);
   }
```

Comment:
Oauth login page offers the user to make a new account and reset password etc...